### PR TITLE
Remove attempt to Rayleigh correct the VIIRS I02 band 

### PR DESCRIPTION
**Pull request by adybbroe**
Remove attempt to Rayleigh correct the VIIRS I02 band for the false-color RGB. The band is beyond 0.8 micron and there is effectively no rayleigh scattering in this band. Pyspectral cannot correct this anyhow.

We get warning messages in our production, like this:
`WARNING  pyspectral.rayleigh] Effective wavelength for band I02 outside nominal 400-800 nm range!`

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Fully documented 
 - [ ] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Comment by adybbroe** [comment]
Here is the spectral response function of I2 for reference:

rsr_band_I2

**Review by mraspaud — APPROVED** [review]
LGTM!

**Comment by djhoese** [comment]
I've updated the title to reflect that this is for the false color RGB specifically.

File: satpy/etc/composites/viirs.yaml [hunk 1]
```diff
@@ -232,7 +232,7 @@ composites:
       modifiers: [sunz_corrected, rayleigh_corrected]
     optional_prerequisites:
     - name: I02
-      modifiers: [sunz_corrected_iband, rayleigh_corrected_iband]
```
Comment (djhoese) [inline satpy/etc/composites/viirs.yaml:235]: Should rayleigh be removed from the M05 band too? I suppose that is still in the "correctable" range?

**Pull request merged**